src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -3995,6 +3995,9 @@ namespace WaterToAirHeatPumpSimple {
         auto &wahp = state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil);
         if (wahp.WAHPType == WatertoAirHP::Cooling) {
             if (wahp.RatedEntAirWetbulbTemp != DataSizing::AutoSize) {
+                if (wahp.TotalCoolCapCurve == nullptr || wahp.CoolPowCurve == nullptr) {
+                    return;
+                }
```

Comment:
Protect calls to curves with null pointers (i.e., misspelled curve names or missing curve objects)